<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Contact</class>
 <widget class="QWidget" name="Contact">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="categoryWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>420</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(40, 46, 51);</string>
   </property>
   <widget class="QPushButton" name="cancelButon">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>7</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:  #ff006c;
 border-style: solid;
 border-radius:15px;
 max-width:30px;
 max-height:30px;
 min-width:30px;
 min-height:30px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../../../Рабочий стол/ClientRes/arrow_white.png</normaloff>../../../../../Рабочий стол/ClientRes/arrow_white.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="newContactLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>5</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Контакт</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteButon">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>7</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:  #ff006c;
 border-style: solid;
 border-radius:15px;
 max-width:30px;
 max-height:30px;
 min-width:30px;
 min-height:30px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../../../Рабочий стол/ClientRes/delete_white.png</normaloff>../../../../../Рабочий стол/ClientRes/delete_white.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="editButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>7</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:  #ff006c;
 border-style: solid;
 border-radius:15px;
 max-width:30px;
 max-height:30px;
 min-width:30px;
 min-height:30px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../../../Рабочий стол/ClientRes/create_white.png</normaloff>../../../../../Рабочий стол/ClientRes/create_white.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="titleHint_1" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>400</width>
     <height>16</height>
    </rect>
   </property>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>350</width>
      <height>1</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>15</width>
      <height>1</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>55</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Имя</string>
    </property>
   </widget>
  </widget>
  <widget class="QPlainTextEdit" name="contentPlainTextEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>400</width>
     <height>181</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPlainTextEdit{
	background-color: rgb(35, 38, 41); 
	selection-background-color: rgb(43,45,56);  
	border-style: solid;
	color:  #FFF; 
	border-radius: 10px; 
	padding-left: 20px;
	padding-right: 20px; 
	padding-top:5px;
	padding-bottom:5px;
}


QScrollBar:vertical
 {
     background-color: #2A2929;
     width: 12px;
     margin: 15px 3px 15px 3px;
     border: 1px transparent #2A2929;
     border-radius: 3px;
 }

 QScrollBar::handle:vertical
 {
     background-color:  #ff006c;
     min-height: 5px;
     border-radius: 3px;
 }

 QScrollBar::sub-line:vertical
 {
     margin: 3px 0px 3px 0px;
     border-image: url(:/images/up_arrow_disabled.png);        /* # &lt;-------- */
     height: 10px;
     width: 10px;
     subcontrol-position: top;
     subcontrol-origin: margin;
 }

 QScrollBar::add-line:vertical
 {
     margin: 3px 0px 3px 0px;
     border-image: url(:/images/down_arrow_disabled.png);       /* # &lt;-------- */
     height: 10px;
     width: 10px;
     subcontrol-position: bottom;
     subcontrol-origin: margin;
 }

 QScrollBar::sub-line:vertical:hover,QScrollBar::sub-line:vertical:on
 {

     border-image: url(:/images/up_arrow.png);                  /* # &lt;-------- */
     height: 10px;
     width: 10px;
     subcontrol-position: top;
     subcontrol-origin: margin;
 }


 QScrollBar::add-line:vertical:hover, QScrollBar::add-line:vertical:on
 {
     border-image: url(:/images/down_arrow.png);                /* # &lt;-------- */
     height: 10px;
     width: 10px;
     subcontrol-position: bottom;
     subcontrol-origin: margin;
 }

 QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical
 {
     background: none;
 }


 QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical
 {
     background: none;
 }</string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="titleHint_3" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>400</width>
     <height>16</height>
    </rect>
   </property>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>320</width>
      <height>1</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>15</width>
      <height>1</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>81</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Комментарий</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="titleHint_2" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>400</width>
     <height>16</height>
    </rect>
   </property>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>320</width>
      <height>1</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>15</width>
      <height>1</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>55</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Компания</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="titleHint_4" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>400</width>
     <height>16</height>
    </rect>
   </property>
   <widget class="Line" name="line_7">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>330</width>
      <height>1</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_8">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>15</width>
      <height>1</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>55</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Телефон</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="titleHint_5" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>400</width>
     <height>16</height>
    </rect>
   </property>
   <widget class="Line" name="line_9">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>270</width>
      <height>1</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_10">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>15</width>
      <height>1</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>111</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Электронная почта</string>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="phoneLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>400</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: solid;
border-radius: 10px; 

color:  #FFF; 
background-color: rgb(35, 38, 41); 
selection-background-color: rgb(43,45,56);  


padding-left: 10px;
padding-right: 10px; 
padding-top:5px;
padding-bottom:5px;</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="emailLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>400</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: solid;
border-radius: 10px; 

color:  #FFF; 
background-color: rgb(35, 38, 41); 
selection-background-color: rgb(43,45,56);  


padding-left: 10px;
padding-right: 10px; 
padding-top:5px;
padding-bottom:5px;</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="companyLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>400</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: solid;
border-radius: 10px; 

color:  #FFF; 
background-color: rgb(35, 38, 41); 
selection-background-color: rgb(43,45,56);  


padding-left: 10px;
padding-right: 10px; 
padding-top:5px;
padding-bottom:5px;</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>400</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: solid;
border-radius: 10px; 

color:  #FFF; 
background-color: rgb(35, 38, 41); 
selection-background-color: rgb(43,45,56);  


padding-left: 10px;
padding-right: 10px; 
padding-top:5px;
padding-bottom:5px;</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
